app/assets/stylesheets/components/form/form.scss

```diff
@@ -154,3 +158,11 @@
     }
   }
 }
+
+@media (max-width: 405px){
+  .form__section.form__section--bottom > .button {   
```

Comment:
As discussed in discord we should have a wrapper div around the buttons instead.